packages/edit-navigation/src/components/menu-selector/index.js

```diff
@@ -0,0 +1,58 @@
+/**
+ * WordPress dependencies
+ */
+import {
+	MenuGroup,
+	MenuItemsChoice,
+	Button,
+	Dropdown,
+} from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+/**
+ * Internal dependencies
+ */
+import AddMenu from '../add-menu';
+
+export default function MenuSelector( { onSelectMenu, menus } ) {
+	return (
+		<div className="edit-navigation-menu-selector-wrapper">
+			<div className="edit-navigation-menu-selector-header">
+				{ __( 'Chose a menu to edit: ' ) }
+			</div>
+			<div className="edit-navigation-menu-selector">
+				<div className="editor-styles-wrapper">
+					<MenuGroup>
+						<MenuItemsChoice
+							onSelect={ onSelectMenu }
+							choices={ menus.map( ( menu ) => ( {
+								value: menu.id,
+								label: menu.name,
+							} ) ) }
+						/>
+					</MenuGroup>
```

Comment:
A MenuGroup should generally be in a `NavigableMenu`. `NavigableMenu` provides arrow key navigation between menu items and has the correct `role` of `menu`.